components/server/src/user/user-controller.ts

```diff
@@ -190,38 +190,6 @@ export class UserController {
             }
         });
 
-        router.get(
```

Comment:
How do we login to a websession with the new credentials? Not set up, yet?

Reply:
It's implemented above this diff - https://github.com/gitpod-io/gitpod/pull/16761/files/0ce214c49a96e782b23f6079c873f01218700d75#diff-0fda9115576b7e1168858f5d4f8869e3120c3a8364cc8f43335037ebb78f6734L145